src/export2mpb.jl

```diff
@@ -58,7 +58,7 @@ function _mat2matrix3x3(m::SMatrix{D,D,<:Real}) where D
         m′[1,1] = only(m)
         m = SMatrix(m′)
     elseif D == 2 # extend 2D to 3D
-        m = hcat(vcat(m, @SVector[0,0]'), @SVector[0,0,1])
+        m = hcat(vcat(m, @SVector[0,0]), @SVector[0,0,1])
```

Comment:
This won't work; the original version is correct afaict (the new version would throw `ERROR: DimensionMismatch("Tried to vcat arrays of size (2, 2) and (2,)"`)